build.sh

```diff
@@ -11,6 +11,7 @@ usage()
     echo "clean - optional argument to force a clean build."
     echo "verbose - optional argument to enable verbose build output."
     echo "clangx.y - optional argument to build using clang version x.y."
+    echo "installcli - should download CLI and not use cached copy of the CLI."
```

Comment:
When would one use this option? There are other parts that the build scripts download - we do not have options for those either.